### PR TITLE
Fixes #2 - Adds a second screen for the template generator

### LINKED ISSUE #2: Add a second screen for the template generator
This includes building the ui for the second screen of the generator without the template preview functionality (it will be addressed in https://github.com/webcompat/outreach/issues/4).

We can preselect dropdowns options to be (English -> Email)

Prototype for mobile:

Screen Shot 2020-07-02 at 12 52 31 PM

And desktop:
Screen Shot 2020-07-02 at 1 16 54 PM

### PR DESCRIPTION
**Pull request by ksy36**
This includes selects for language and template and a template previewer  r? @magsout

### CODE REVIEW

File: src/templates/english/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    },
+    linkedin: {
+      label: "LinkedIn",
+      value: "linkedin",
+      template: "LinkedIn template",
+    },
+  },
+  label: "English",
+  value: "english",
+};
```
Comment (ksy36) [inline src/templates/english/index.js:24]: I've added a couple templates, for English and Russian. Some of the are just placeholders for now, I'll add real text later

File: src/components/Previewer.svelte [hunk 2]
```diff
@@ -1,15 +1,34 @@
...
+  export let selectedTemplate;
+  export let issue;
+
+  const getTemplate = (languages, lng, tpl) => {
+    if (lng && tpl) {
+      return languages[lng].templates[tpl];
+    }
+  };
+
+  $: tpl = getTemplate(languages, selectedLanguage, selectedTemplate) || {};
```
Comment (magsout) [inline src/components/Previewer.svelte:15]: it's svelte syntax ? ` $:`

**Review by magsout — APPROVED** [review]
very nice. I could help for the french template if needed

sorry for the delay for this review

File: src/components/Previewer.svelte (hunk 2 above)
Comment (ksy36) [reply src/components/Previewer.svelte:15]: yeah, this statement will rerun if one of the values changes https://svelte.dev/examples#reactive-declarations

**Comment by ksy36** [comment]
Thanks!

**Pull request merged**